Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -157,25 +166,40 @@
         field,
         CKFPerformanceConfig(
             ptMin=1.0 * u.GeV if ttbar_pu200 else 0.0,
-            nMeasurementsMin=6,
+            nMeasurementsMin=7,
         ),
         TrackSelectorRanges(
             pt=(1.0 * u.GeV, None),
             absEta=(None, 3.0),
             loc0=(-4.0 * u.mm, 4.0 * u.mm),
         ),
         outputDirRoot=outputDir,
+        outputDirCsv=outputDir,
```

Comment:
is this relevant for training? otherwise I would just drop it since we are writing out the root files as well

same fore the ambiguity below

Reply:
The training uses the csv files as input so yes it is kind of needed. The script I provide for performance evaluation also use Csv so it is also useful to have it for the solver. (They should not be too big ~ 0.1 to 1 Mo per event)

Reply:
I think we should stick with root and for the training people can just trow in csv directories by hand. because the full chain is just meant for sim+reco I would argue

Reply:
I don't care that much, but I am not sure I follow your reasoning, why would root be better than Csv for sim+reco ? They are many people that use the Csv files for performance analysis afterward. If the concern is disk space it doesn't really matter as for 100 ttbar event the Csv represent less than 1% of the output.
If you insist on removing it I would argue it should at least be commented out so people know they can change this option.

Reply:
I also don't really care but I would like to keep it consistent. so either write everything in root or csv or both.

the only benefit of using root is that you get a single file and the writers tend to be more complete. at least from my experience sometimes the csv output does not contain all the information the root files do

Reply:
So what I propose is that for all algorithm the let you write to csv I add  `// outputDirCsv=outputDir,` commented out that way it is still consistent and if people try to do stuff that require the Csv they should realise what they need to do to get then csv files without having to ask us directly. Works for you ?

Reply:
sounds good to me :+1: